lib/katello/engine.rb

```diff
@@ -26,6 +26,7 @@ class Engine < ::Rails::Engine
         :redhat_repository_url => 'https://cdn.redhat.com',
         :consumer_cert_rpm => 'katello-ca-consumer-latest.noarch.rpm',
         :consumer_cert_sh => 'katello-rhsm-consumer',
+        :event_monitor_enabled => true,
```

Comment:
Where is this setting used?

Reply:
I think I refactored its usage away but I'd like to add it back. Never know when that knob may be handy.